assets/src/data/index.js

```diff
@@ -0,0 +1,5 @@
+/**
+ * Register stores
+ */
+import './eventespresso/core';
+import './eventespresso/lists';
```

Comment:
This is necessary to ensure that the `default` export from each of these locations is executed by whatever script imports this file.  Currently nothing is doing so, I think I may introduce a new built file explicitly for eventespresso/data which will depend on the `eejs-core` script.  Then `eventespresso/data` (or whatever the script handle ends up being for this built file) will automatically be setup for any script that needs it.  I think this can be done in a separate pull request once we start implementing blocks.

Reply:
what we be the benefit of doing that over what is here now?

Reply:
I'm sorry, even with your edit I'm not sure I understand exactly what you are asking.  Specifically I don't know what "what is here now?" is referring to.  Is it referring to the lack of this file in the Gutenberg/master branch?

Right now, the stores are not actually registered anywhere (although there is the code in place _to register_, that code is not actually _executed_ anywhere).  In order to regsiter the stores, `core/index.js` and `lists/index.js` would need to be invoked.  This file is prep work for that, all I'd do is add a configuration to the webpack file to build js file with the store registration in them, and expose it (likely on `eejs.stores`) and then ensure that file is loaded via a dependency on any built javascript requiring them.

Does that help or was there something else you were needing to know?